src/harness/parallel/worker.ts

```diff
@@ -55,7 +55,7 @@ namespace Harness.Parallel.Worker {
             retries() { return this; },
             slow() { return this; },
             timeout(n) {
-                timeout = n;
+                timeout = n as number;

```

Comment:
should this be `+n` instead?

Reply:
Depends on whether we believe it to be a number already, or believe that it actually could be a string and want to convert it to a number at runtime. @weswigham might know if the new type definition is actually accurate to our situation